src/frontend/src/components/ActionsMenu.tsx

```diff
@@ -36,22 +61,16 @@ const ActionsMenu: React.FC<ActionsMenuProps> = ({ buttons }) => {
         id="reimbursement-request-actions-dropdown"
         onClick={handleClick}
       >
-        Actions
+        {title}
       </NERButton>
       <Menu open={dropdownOpen} anchorEl={anchorEl} onClose={handleDropdownClose}>
-        {buttons.map((button, index) => (
-          <MenuItem
-            key={index}
-            onClick={() => {
-              handleDropdownClose();
-              button.onClick();
-            }}
-            disabled={button.disabled}
-          >
-            <ListItemIcon>{button.icon}</ListItemIcon>
-            {button.title}
-          </MenuItem>
-        ))}
+        {buttons.map((button, index) => {
+          return index === buttons.length - 1 ? (
```

Comment:
oh ok I see how you did this, I think it'd be better to have a dividerTop? prop on the ButtonInfo type instead, so that a dev can specify which button they want the divider on top of. Should make it so you can delete the MenuButton component as well bc it makes this file more complex than it needs to be